### PR TITLE
fix: Updating step 5 in 'Add a Linked Resource' topic

### PR DESCRIPTION
**Pull request by guipontes7**
## Description of changes
Hi!

In this PR, only one change was made:
- The "..." button has been replaced by the "SETTINGS" button

## GitHub issues resolved by this PR

This PR solves #848

### CODE REVIEW

File: docs/linked-resources/add-linked-resource.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ links:
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
 1. Click `DETAILS` on the application you want to link a server
-1. On the upper-right corner of the application details view, click `...`
+1. On the upper-right corner of the application details view, click `SETTINGS`
```
Comment (jfoliveira) [inline docs/linked-resources/add-linked-resource.md:24]: @guipontes7 thanks for updating the docs with this change.
Could you update this PR applying the same change in all the references of "click `...`"?

For a full list of files containing the outdated text, see: https://github.com/search?q=repo%3Adevopness%2Fdevopness%20click%20%60...%60&type=code

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/linked-resources/add-linked-resource.md (hunk 1 above)
Comment (guipontes7) [reply docs/linked-resources/add-linked-resource.md:24]: Yes!

PR updated.

File: docs/files/edit-file.md [hunk 2]
```diff
@@ -20,8 +20,7 @@ required_permissions:
 1. On the upper-right corner of the applications details view click `SETTINGS`
 1. Use the drop-down menu to choose `CONFIGURATION FILES`
 1. In the list of configuration files find the configuration file you want to edit and click `DETAILS`
-1. On the upper-right corner of the key details click `...`
-1. Use the drop-down menu to choose `EDIT`
+1. On the upper-right corner of the key details click `EDIT`
```
Comment (thlmenezes) [inline docs/files/edit-file.md:23]: @guipontes7 could you apply changes below to help us with consistent use of commas on this type of instruction?

```suggestion
1. On the upper-right corner of the key details, click `EDIT`
```

File: docs/applications/edit-application.md [hunk 3]
```diff
@@ -17,8 +17,7 @@ required_permissions:
 1. Find the `Applications` card
 1. Click `View` in the `Applications` card to see a list of existing `Applications`
 1. Click `DETAILS` on the application you want to edit
-1. On the upper right corner of the application details view click `...`
-1. Use the drop-down menu to choose `EDIT`
+1. On the upper right corner of the application details view click `EDIT`
```
Comment (thlmenezes) [inline docs/applications/edit-application.md:20]: @guipontes7 could you apply changes below to help us with consistent use of commas on this type of instruction?

```suggestion
1. On the upper right corner of the application details view, click `EDIT`
```

File: docs/pipelines/edit-pipeline.md [hunk 4]
```diff
@@ -20,8 +20,7 @@ required_permissions:
 1. On the upper-right corner of the applications details view click `SETTINGS`
 1. Use the drop-down menu to choose `PIPELINES`
 1. In the list of pipelines find the pipeline you want to edit and click `DETAILS`
-1. On the upper-right corner of the key details click `...`
-1. Use the drop-down menu to choose `EDIT`
+1. On the upper-right corner of the key details click `EDIT`
```
Comment (thlmenezes) [inline docs/pipelines/edit-pipeline.md:23]: @guipontes7 could you update this PR applying the changes below ? I believe this is more aligned with the article objective of editing a pipeline

```suggestion
1. On the upper-right corner of the pipeline details view, click `EDIT`
```

**Review by thlmenezes — CHANGES_REQUESTED** [review]

File: docs/pipelines/edit-pipeline.md (hunk 4 above)
Comment (thlmenezes) [reply docs/pipelines/edit-pipeline.md:23]: Just clarifying, this was a bug that i noticed while reviewing your PR but not caused by your changes in any way.

File: docs/applications/edit-application.md (hunk 3 above)
Comment (guipontes7) [reply docs/applications/edit-application.md:20]: File updated!

File: docs/pipelines/edit-pipeline.md (hunk 4 above)
Comment (guipontes7) [reply docs/pipelines/edit-pipeline.md:23]: File updated!

**Comment by jfoliveira** [comment]
@cla-bot check

**Review by jfoliveira — APPROVED** [review]

**Review by thlmenezes — APPROVED** [review]

**Pull request closed without merging**